packages/lib/src/interactions/AxialSelectionTool.tsx

```diff
@@ -31,21 +31,21 @@ function AxialSelectionTool(props: Props) {
       context
     );
 
-    const { startPoint: mouseStartPoint, endPoint: mouseEndPoint } = selection;
```

Comment:
`mouse` made it sound like it was a point in HTML space. I went with renaming the other variables to `axialStartPoint` and `axialEndPoint`. Let me know what you think.